src/stories/SpeedInput.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { ComponentPropsWithoutRef } from 'react';
```

Comment:
The SpeedInput.Stories.tsx file should be inside the SpeedInput folder so that the path would be. "src/stories/SpeedInput"